spec/system/kudo_system_spec.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Kudo management', type: :system do
+  before do
+    driven_by(:rack_test)
+  end
+
+  # rubocop:disable RSpec/ExampleLength
+
+  it 'enables me to create, edit and destroy kudos' do
+    FactoryBot.create :employee,  email: 'test3@test.pl',
+                                  password: 'zaq123',
+                                  password_confirmation: 'zaq123'
+
+    visit root_path
+    click_on 'Log in'
+    fill_in 'employee[email]', with: 'test3@test.pl'
+    fill_in 'employee[password]', with: 'zaq123'
+    click_button 'Log in'
+
+    visit '/kudos/new'
+    fill_in 'kudo[title]', with: 'Test Title of Kudo'
+    fill_in 'kudo[content]', with: 'Test Content '
+    select 'test3@test.pl'
+    click_button 'Create Kudo'
+
+    expect(page).to have_text('Kudo was successfully created.')
+
+    visit '/kudos'
+    click_on 'Edit...'
+    fill_in 'kudo[title]', with: 'Edited Test Title of Kudo'
+    click_button 'Update Kudo'
+
+    expect(page).to have_text('Kudo was successfully updated.')
+
+    visit '/kudos'
+    click_on 'Destroy!'
+
+    expect(page).to have_text('Kudo was successfully destroyed.')
+  end
+
+  # rubocop:enable RSpec/ExampleLength
+end
```

Comment:
Here, I would also use the kudo factory. In addition, I would introduce an additional check to see if your kudo has actually been saved correctly, the easiest way is to check if the content of the title has appeared on the page

Reply:
Checking creation of Kudo via factory - added!